default.nix

```diff
@@ -14,6 +14,8 @@ in
     inherit buildInputs;
 
     buildPhase = ''
+      # We can't check external links inside the sandbox, but it's good to check them outside the sandbox.
+      sed -i 's/follow-web-links = true/follow-web-links = false/g' book.toml
```

Comment:
Use `substituteInPlace --replace-fail` to make it fail loudly if the `book.toml` is changed.

Reply:
Done! I actually looked for a way to make sed fail if it doesn't find the pattern, but it wasn't trivial so I let it go.